packages/icons/src/k8s-rollout.svg

```diff
@@ -0,0 +1 @@
3 1.93 0 0 0 .205-.079 2.14 2.14 0 0 1 .399-.117c.047.003.093.019.131.047l.1-.017a2.98 2.98 0 0 0 1.32 1.648l-.042.1a.23.23 0 0 1 .02.129 2.29 2.29 0 0 1-.207.372 1.92 1.92 0 0 0-.123.183l-.029.062a.23.23 0 0 0 .023.269c.021.025.047.044.076.058a.23.23 0 0 0 .093.022c.032.001.064-.005.094-.017a.23.23 0 0 0 .129-.135h0 0l.028-.058c.026-.069.048-.139.066-.21a1.6 1.6 0 0 1 .177-.411c.027-.026.062-.042.099-.048l.052-.094a2.96 2.96 0 0 0 2.112.005l.049.088c.047.008.089.033.117.071.065.123.117.252.157.385a1.9 1.9 0 0 0 .066.21l.028.059a.23.23 0 0 0 .128.135c.03.012.062.018.094.017a.23.23 0 0 0 .223-.258.23.23 0 0 0-.031-.09l-.03-.062a1.83 1.83 0 0 0-.123-.183c-.079-.114-.147-.236-.203-.363a.18.18 0 0 1 .018-.135.81.81 0 0 1-.038-.092 2.98 2.98 0 0 0 1.319-1.66l.098.017c.035-.032.081-.049.128-.048.137.026.27.065.399.117.067.03.136.057.205.079l.059.013.008.002a.23.23 0 0 0 .296-.167.23.23 0 0 0-.193-.279zm-2.175-2.332l-.978.694-.003-.002c-.03.021-.064.035-.101.038s-.073-.003-.106-.019-.061-.04-.081-.071-.032-.066-.033-.102h-.001l-.068-1.199a2.38 2.38 0 0 1 1.372.661zm-1.843 1.314h.368l.229.286-.082.357-.331.159-.332-.159-.082-.357.23-.286zm-.346-1.934l.244-.042-.068 1.201-.005.002c-.002.036-.013.072-.033.102a.2.2 0 0 1-.081.071c-.033.016-.069.023-.106.019s-.071-.016-.101-.038l-.002.001-.985-.698a2.36 2.36 0 0 1 1.136-.619zm-1.493 1.067l.899.804-.001.005c.027.024.048.055.059.09s.013.072.005.108-.026.068-.051.095a.2.2 0 0 1-.092.055l-.001.004-1.152.332c-.057-.521.06-1.046.335-1.492zm1.204 2.136l-.458 1.106a2.37 2.37 0 0 1-.951-1.19l1.181-.201.002.003c.013-.003.026-.004.04-.003.033 0 .066.009.095.025s.054.039.072.067.029.06.032.093-.003.066-.016.097l.003.004zm1.351 1.423c-.428.098-.876.075-1.292-.066l.581-1.051h.001c.017-.032.043-.059.074-.078s.067-.029.104-.029a.21.21 0 0 1 .104.028c.032.018.057.046.074.078h.004l.583 1.052-.233.065zm.751-.312l-.463-1.118.001-.002a.2.2 0 0 1-.004-.15c.018-.048.055-.089.101-.111.024-.012.05-.019.077-.02.016-.001.031.001.046.004l.002-.003 1.192.201a2.36 2.36 0 0 1-.954 1.199zm1.079-1.758l-1.158-.334-.001-.005c-.035-.01-.067-.029-.092-.055a.2.2 0 0 1-.051-.095c-.008-.036-.006-.073.005-.107s.032-.066.059-.09v-.002l.894-.8a2.4 2.4 0 0 1 .345 1.488z" fill="#fff"/><g fill-rule="evenodd"><path d="M9.014.5a.5.5 0 0 0-.51 0L.245 5.397a.5.5 0 0 0-.245.43v9.434a.5.5 0 0 0 .243.429l8.647 5.169a.5.5 0 0 0 .511.001l8.353-4.933a.5.5 0 0 0 .246-.431v-3.673a.5.5 0 1 0-1 0v3.388l-7.347 4.339.056-8.263 3.901-2.304a.5.5 0 1 0-.508-.861l-3.89 2.297L1.48 5.828l7.28-4.316 4.115 2.437a.5.5 0 1 0 .509-.86L9.014.5zm-.305 10.784l-.056 8.268L1 14.978V6.706l7.709 4.579z" fill="#4f5162"/><path d="M17.5 4.906c-1.433 0-2.594 1.161-2.594 2.594s1.161 2.594 2.594 2.594 2.594-1.161 2.594-2.594a.5.5 0 1 1 1 0c0 1.985-1.609 3.594-3.594 3.594S13.906 9.485 13.906 7.5s1.609-3.594 3.594-3.594a3.58 3.58 0 0 1 2.39.91V4.64a.5.5 0 1 1 1 0v1.402a.5.5 0 0 1-.399.494c-.033.007-.066.01-.101.01h-1.406a.5.5 0 1 1 0-1h.222a2.58 2.58 0 0 0-1.706-.64z" fill="#0278d5"/></g></svg>
```

Comment:
Pls make sure svgs are following guidelines mentioned in this doc : https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices